.eslintrc

```diff
@@ -48,7 +48,13 @@
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-var-requires": "off",
     "@typescript-eslint/no-useless-constructor": "warn",
-    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/no-unused-vars": [
+      "error", {
+         "argsIgnorePattern": "^_",
+         "varsIgnorePattern": "^_",
+         "caughtErrorsIgnorePattern": "^_"
+    }],
+    "no-unused-vars": "off",
```

Comment:
does this mean they that it will still cause the linter to fail if the variable does not start with `_`?

Reply:
yeah, unused variables would have to start with _